garrysmod/gamemodes/terrortown/gamemode/init.lua

```diff
@@ -889,7 +889,6 @@ function SelectRoles()
    -- traitor, so becoming detective does not mean you lost a chance to be
    -- traitor
    local ds = 0
-   local min_karma = GetConVar("ttt_detective_karma_min"):GetInt() or 0

```

Comment:
Keep this cached outside the loop so it doesn't get called multiple times uselessly.

Reply:
> Keep this cached outside the loop so it doesn't get called multiple times uselessly.

?